lib/presentation/shared_widgets/commitment_card.dart

```diff
@@ -0,0 +1,170 @@
+import 'package:flutter/material.dart';
+
+import '../../domain/crowdaction/commitment.dart';
+import '../../presentation/themes/constants.dart';
+
+class CommitmentCardList extends StatelessWidget {
+  final List<Commitment> commitments;
+  final Function(int) onSelected;
+  final Axis axis;
+
+  /// Widget for easily creating a list of CommitmentCard(s)
+  ///
+  ///[commitments] The commitments the list is made up of
+  ///
+  /// [onSelected] Callback function for when a card is selected,
+  /// returns the id of the selected commitment
+  ///
+  /// [axis] The direction of the list, defaults to [Axis.vertical]
+  const CommitmentCardList({
```

Comment:
Can't a ListView builder replace this whole widget?

Reply:
@lorenzowijtman Could you elaborate on this